soh/src/overlays/actors/ovl_En_Ru2/z_en_ru2.h

```diff
@@ -30,4 +30,6 @@ typedef struct EnRu2 {
     /* 0x02C8 */ s16 subCamId;
 } EnRu2; // size = 0x0314
 
+void func_80AF36EC(EnRu2* enRu2, PlayState* play);
```

Comment:
I know the original Vanilla Behavior PR showed examples of adding declarations to actor header files for use in .cpp code, but that is not strictly required. Instead we would like to avoid this in favor of foward declaring them in extern blocks in the .cpp file (refer to hook_handlers.cpp for what I mean)

```c++
extern "C" {
...
extern void func_80AF36EC(EnRu2* enRu2, PlayState* play);
}
```

This recommendation applies for all new code additions. We will have a cleanup effort to try and revert those header changes from previous VB additions.